containers/daap-presigned-url/src/var/task/presigned_url.py

```diff
@@ -1,15 +1,20 @@
 import json
-import logging
 import os
 import uuid
 from datetime import datetime
 
 import boto3
+from data_platform_logging import DataPlatformLogger
 
-logging.basicConfig(level=logging.INFO, force=True)
-root_logger = logging.getLogger()
 s3 = boto3.client("s3")
 
+logger = DataPlatformLogger(
+    extra={
+        "image_version": os.getenv("VERSION", "unknown"),
+        "base_image_version": os.getenv("BASE_VERSION", "unknown"),
+    }
+)
+
 
 def handler(event, context):
     bucket_name = os.environ["BUCKET_NAME"]
```

Comment:
after L20 you can log inputs from queryStringParameters aka the only ones which will change from execution to executipn